### PR TITLE
custom pages example and an email adapter is added

### PR DESCRIPTION
**Pull request by macarthuror**
Added to the README file.

Added an example of how to change the pages for the ones who didn't know how to do it and a link for an email adapter with multi language template if anyone need it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,6 +289,7 @@ var server = ParseServer({
 ```
 
 You can also use other email adapters contributed by the community such as:
+- [parse-smtp-template (Multi Languaje and Multi Template)](https://github.com/macarthuror/parse-smtp-template)
```
Comment (davimacedo) [inline README.md]: Can you add the npm link instead?

**Comment by davimacedo** [comment]
@TomWFox do you want to take a look as well?

File: README.md (hunk 1 above)
Comment (macarthuror) [reply README.md]: Sure 👌

File: README.md [hunk 2]
```diff
@@ -298,6 +300,27 @@ You can also use other email adapters contributed by the community such as:
 - [simple-parse-smtp-adapter](https://www.npmjs.com/package/simple-parse-smtp-adapter)
 - [parse-server-generic-email-adapter](https://www.npmjs.com/package/parse-server-generic-email-adapter)
 
+### Custom Pages
+
+Parse server can let you  change the default pages of the app and redirect the user to other path or domain.
```
Comment (TomWFox) [inline README.md]:
```suggestion
It’s possible to change the default pages of the app and redirect the user to another path or domain.
```

File: README.md [hunk 3]
```diff
@@ -288,6 +289,7 @@ var server = ParseServer({
 ```
 
 You can also use other email adapters contributed by the community such as:
+- [parse-smtp-template (Multi Languaje and Multi Template)](https://www.npmjs.com/package/parse-smtp-template)
```
Comment (TomWFox) [inline README.md]:
```suggestion
- [parse-smtp-template (Multi Language and Multi Template)](https://www.npmjs.com/package/parse-smtp-template)
```

Comment (macarthuror) [reply README.md]: Thanks :)

**Pull request merged**